examples/load-data-via-graph-construction.ipynb

```diff
@@ -0,0 +1,326 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "view-in-github"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/nvitucci/graph-data-science-client/blob/add-construct-notebook/examples/load-data-via-graph-construction.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```

Comment:
Is this colab source relevant?

Reply:
You are right, it should point to this repo instead (it points to mine because we are using a fork). Let me try and fix it.